### ISSUE TITLE
large image with a significant amount of data on this page

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2532]
## Source

https://airtable.com/appRFUc45sF4yWFut/tbl51inE74gEWlB0x/viwG9G4vAjsjQEh4g/recVvMpG8q2lClM4w?blocks=hide

## Describe the bug

There is a large image with a significant amount of data on this page, but it has no alt text and the text below does not appear to cover the information contained in the images of text. 	

https://www.adapacific.org/casp-exam-preparation-course/

## Recommendations

Provide alt text and include the available information in images of text in the text on this page or via a link.

**Comment by curtismchale** [comment]
@lukefretwell that's not really us as the client should be adding ALT text to describe the image. I did check and any text they set up for the image shows up in this case so there isn't anything for us to do IMO.

**Comment by lukefretwell** [comment]
@curtismchale yes, I've created a `content` label to indicate what Pacific ADA can address through the CMS.

**Comment by curtismchale** [comment]
@kevindherman @lukefretwell I'm assuming this was communicated with the client and closing the issue.
